File: TUGAS DPBO/Latihan 1/B/C++/club.cpp
#include <iostream>
#include <string>

using namespace std;

class club{
    private:
    string name;
    string country;
    string year;
    string player;
    string coach;

    public:
    club(){
        this->name = "";
        this->country = "";
        this->year = "";
        this->player = "";
        this->coach = "";
    }

    club(string name, string country, string year, string player, string coach){
        this->name = name;
        this->country = country;
        this->year = year;
        this->player = player;
        this->coach = coach;
    }

    void setName(string name){
        name = name;
    }

    void setCountry(string country){
        this->country = country;
    }

    void setYear(string year){
        this->year = year;
    }

    void setPlayer(string player){
        this->player = player;
    }

    void setCoach(string coach){
        this->coach = coach;
    }

    string getName(){
        return this->name;
    }

    string getCountry(){
        return this->country;
    }

    string getYear(){
        return this->year;
    }

    string getPlayer(){
        return this->player;
    }

    string getCoach(){
        return this->coach;
    }

    ~club(){}
};

File: TUGAS DPBO/Latihan 1/B/java/club.java
public class club {
    private String name;
    private String country;
    private String year;
    private String player;
    private String coach; 

    public club(String name, String country, String year, String player, String coach){
        this.name = name;
        this.country = country;
        this.year = year;
        this.player = player;
        this.coach = coach;
    }

    //methode setter
    public void setName(String name){
        this.name = name;
    }

    public void setCountry(String country){
        this.country = country;
    }

    public void setYear(String year){
        this.year = year;
    }

    public void setPlayer(String player){
        this.player = player;
    }

    public void setCoach(String coach){
        this.coach = coach;
    }


    public String getName(){
        return name;
    }

    public String getCountry(){
        return country;
    }

    public String getYear(){
        return year;
    }

    public String getPlayer(){
        return player;
    }

    public String getCoach(){
        return coach;
    }
}
